test/mobile/test_bytecode.py

```diff
@@ -264,5 +297,12 @@ def test_backport_bytecode_from_file_to_buffer(self):
             torch.testing.assert_allclose(mobile_module_result, expected_mobile_module_result)
 
 
+    def test_get_model_ops_and_info(self):
```

Comment:
can we just wait for v6 to go in? That way hardcoding this test would not be needed (the test could then be about models v6 and above to return #args > 0).